Automattic/wp-calypso
# Plans overhaul Phase 1: Apply latest copy and design to Plans pages

#### Changes proposed in this Pull Request
More info: pdgrnI-pJ#comment-768-p2

* adds the latest copy to the overhauled Plan grid
* enables new Plan grid for Atomic sites
* adjusts mobile layout sizes 

<img width="320" alt="Planning page i1 - collapsed - final copy VERSION2" src="https://user-images.githubusercontent.com/2749938/158811260-df11a451-4bb3-4ee8-8faf-68c8f5a90d0b.png">


#### Testing instructions

* Go to /plans/[site]/?flags=plans/pro-plan
* The Plans grid copy should match the design from the description
* Feature tooltip popup text should match the ones from the spreadsheet: 2bf68-pb
<img width="320" alt="Screenshot 2022-03-17 at 14 49 34" src="https://user-images.githubusercontent.com/2749938/158811979-2a169ebd-9a47-4fb0-bbb1-7f0f65d2de6e.png">

<img width="180" alt="Screenshot 2022-03-17 at 14 50 37" src="https://user-images.githubusercontent.com/2749938/158812133-1fd18b83-c619-48af-b02b-4e076d178721.png">

* Go to /start/plans?flags=plans/pro-plan
* The Plans page should match the design from the description

<img width="320" alt="Screenshot 2022-03-17 at 14 52 08" src="https://user-images.githubusercontent.com/2749938/158812459-831ceef8-c398-4da4-a8e0-0ef48471ef80.png">
<img width="180" alt="Screenshot 2022-03-17 at 14 52 23" src="https://user-images.githubusercontent.com/2749938/158812448-05d945ba-9da1-4592-ae1c-2738de74d8f4.png">

* Go to /plans/my-plan/[site]?flags=plans/pro-plan on an Atomic site (ex. Bussiness)
* You should notice the Plan overhaul changes
<img width="320" alt="Screenshot 2022-03-17 at 15 14 03" src="https://user-images.githubusercontent.com/2749938/158816222-4ae35efc-f0f5-40bd-8d87-e9b688133bf6.png">
